ml_git/storages/s3_storage.py

```diff
@@ -51,6 +52,9 @@ def bucket_exists(self):
                 error_msg = output_messages['ERROR_AWS_KEY_NOT_EXIST']
             log.error(error_msg, class_name=STORAGE_FACTORY_CLASS_NAME)
             return False
+        except EndpointConnectionError as e:
+            log.error(output_messages['ERROR_BUCKET_ENDPOINT_CONNECTION'] % (self._bucket, e), class_name=STORAGE_FACTORY_CLASS_NAME)
```

Comment:
Please use `str.format()` here and elsewhere as needed.

```suggestion
            log.error(output_messages['ERROR_BUCKET_ENDPOINT_CONNECTION'].format(self._bucket, e), class_name=STORAGE_FACTORY_CLASS_NAME)
```

Reply:
Fixed on commit [d25bd0](https://github.com/HPInc/ml-git/pull/145/commits/d25bd091a538a0a44d3b78b1b95eb28657c93ae6)